src/views/pages/Start/index.jsx

```diff
@@ -81,7 +82,7 @@ const Start = () => {
         }
         {step === STEPS.ABOUT_PANEL &&
           <AboutPanel
-            onPrevious={previousStep}
+            // onPrevious={previousStep} 
```

Comment:
You can just delete this line instead, git allows us to go back in time if we need to ;)